Anatolie/src/main/java/Numbers.java

```diff
@@ -0,0 +1,7 @@
+public class Numbers {
+    public static void main(String[] agrs) {
+        for (int i = 1; i <= 10; i = i + 1) {
```

Comment:
There is an option to shorten i = i + 1 to i++

Reply:
Done